weaviate/plural/notes.tpl

```diff
@@ -1 +1,5 @@
 Your Weaviate installation is available at https://{{ .Values.hostname }}
+
+Authentication by API-KEY is enabled.
+API-KEY for admin {{ .Values.adminEmail }} is {{ .weaviate.weaviate.admin.key }}
+Make sure to include this header while requesting your API -> Authorization: Bearer {{ .weaviate.weaviate.admin.key }}
```

Comment:
i don't see where `.weaviate.weaviate.admin.key` is set

Reply:
It is set here https://github.com/pluralsh/plural-artifacts/pull/660/files#diff-8ac5632e8e4e6a4f2fd4149769678251e1a889a14ab6a0337598707f48b03ea6R16
And updated dynamically with .tpl file